client/signup/jetpack-connect/index.jsx

```diff
@@ -17,6 +17,13 @@ import Main from 'components/main';
 import JetpackConnectNotices from './jetpack-connect-notices';
 import SiteURLInput from './site-url-input';
 import { dismissUrl, goToRemoteAuth, goToPluginInstall, checkUrl } from 'state/jetpack-connect/actions';
+import versionCompare from 'lib/version-compare';
+
+
+/**
+ * Constants
+ */
+const MINIMUM_JETPACK_VERSION = '3.9.6';
 
```

Comment:
oh! Does the version in jetpack master's readme get bumped after the release? right now it marks 3.9.6 : https://github.com/Automattic/jetpack/blob/master/readme.txt#L4